config/purify.php

```diff
@@ -112,4 +112,25 @@
     //    'cache' => \Stevebauman\Purify\Cache\FilesystemDefinitionCache::class,
     // ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Disable 'purify:clear' command
+    |--------------------------------------------------------------------------
+    |
+    | Here you may enable or disable the 'purify:clear' command.
+    |
+    | If you have configured Purify to utilize the CacheDefinitionCache in the serializer
+    | option, this command issues a Cache::clear() on the cache driver you have configured
+    | it to use.
+    |
+    | If you have configured Purify to utilize the FilesystemDefinitionCache in the serializer
+    | option, this command will clear the directory that you have configured it to store in.
+    | 
+    | You may wish to disable this command if you are not using a unique filesystem path or disk
+    | (via config/filesystems.php) or cache store (via config/cache.php) for Purify.
+    |
+    */
+
+    'disable_clear_command' => false,  
```

Comment:
Hmm I'm not sure I like a boolean config value to disable the command. 

Maybe we should just add a warning (similar to the the default `php artisan migrate` command in production mode), that displays to the executor that it will wipe the entire cache if that driver is configured?

Reply:
Ok no worries. Yeah that would definitely be better than nothing.

I'll probably still disable the command with some `CommandStarting` event hackery on my end.